src/clr-angular/data/datagrid/_variables.datagrid.scss

```diff
@@ -9,6 +9,9 @@ $clr-datagrid-fixed-column-size: (2 * $clr-table-cellpadding) + $clr-table-lineh
 $clr-datagrid-font-color: #adbbc4;
 $clr-datagrid-row-hover: $clr-light-gray;
 $clr-datagrid-action-toggle: $clr-dark-gray;
+$clr-datagrid-pagination-btn-color: #565656;
+$clr-datagrid-pagination-btn-disabled-color: #737373;
```

Comment:
These colors should be in our `$clr-gray` family already. We should use those SASS variables for now.